### PR TITLE
Expand faq.

### PR DESCRIPTION
**Pull request by brian-brazil**
@beorn7 Is that the right plural?

### CODE REVIEW

File: content/docs/introduction/faq.md [hunk 1]
```diff
@@ -30,6 +30,22 @@ The Alertmanager cannot currently be made highly available, but this is a goal.
 
 Most Prometheus components are written in Go. Some are also written in Java and Ruby.
 
+### Why do you pull rather than push?
+
+Pulling over HTTP offers a number of advantages:
+
+- You can run your monitoring on your laptop when developing changes.
```
Comment (juliusv) [inline content/docs/introduction/faq.md]: Let's standardize on `*` for list bullets.

File: content/docs/introduction/faq.md [hunk 2]
```diff
@@ -82,6 +103,10 @@ Yes, using the [Pushgateway](../../instrumenting/pushing). See also the
 [best practices](../../practices/instrumentation/#batch-jobs) for monitoring batch
 jobs.
 
+### What applications can Prometheus monitor?
```
Comment (grobie) [inline content/docs/introduction/faq.md]: `... monitor out of the box?` maybe?

File: content/docs/introduction/faq.md [hunk 3]
```diff
@@ -30,6 +30,22 @@ The Alertmanager cannot currently be made highly available, but this is a goal.
 
 Most Prometheus components are written in Go. Some are also written in Java and Ruby.
 
+### Why do you pull rather than push?
+
+Pulling over HTTP offers a number of advantages:
+
+- You can run your monitoring on your laptop when developing changes.
+- You can more easily tell if a target is down.
+- You can manually go to a target and inspect its health with a web browser.
```
Comment (juliusv) [inline content/docs/introduction/faq.md]: how about another point about rogue targets DDoSing the monitoring server?

File: content/docs/introduction/faq.md [hunk 4]
```diff
@@ -30,6 +30,22 @@ The Alertmanager cannot currently be made highly available, but this is a goal.
...
+
+Overall we believe that push is slightly better than pull, but it should not be
+considered a major point when considering a monitoring system.
+
+The [Push vs Pull for Monitoring](http://www.boxever.com/push-vs-pull-for-monitoring)
+blog post goes into more detail.
```
Comment (juliusv) [inline content/docs/introduction/faq.md]: "...blog post by Boxever goes..."?

File: content/docs/introduction/faq.md (hunk 1 above)
Comment (brian-brazil) [inline content/docs/introduction/faq.md]: Changed them all.

File: content/docs/introduction/faq.md (hunk 3 above)
Comment (brian-brazil) [inline content/docs/introduction/faq.md]: I'm not sure about that, rogue monitoring servers can DDoS the targets.

File: content/docs/introduction/faq.md (hunk 2 above)
Comment (brian-brazil) [inline content/docs/introduction/faq.md]: Done

File: content/docs/introduction/faq.md (hunk 4 above)
Comment (brian-brazil) [inline content/docs/introduction/faq.md]: This is more my opinion than official Boxever opinion. I've put my name in.

File: content/docs/introduction/faq.md [hunk 5]
```diff
@@ -30,6 +30,22 @@ The Alertmanager cannot currently be made highly available, but this is a goal.
...
 
+### Why do you pull rather than push?
+
+Pulling over HTTP offers a number of advantages:
+
+- You can run your monitoring on your laptop when developing changes.
+- You can more easily tell if a target is down.
+- You can manually go to a target and inspect its health with a web browser.
+
+Overall we believe that push is slightly better than pull, but it should not be
```
Comment (juliusv) [inline content/docs/introduction/faq.md]: "pushing is slightly better than pulling" would be more grammatically correct if we embed it into the sentence like this

Comment (beorn7) [inline content/docs/introduction/faq.md]: That should be inverted, right? "Overall, we believe pulling is slightly better than pushing."

File: content/docs/introduction/faq.md [hunk 6]
```diff
@@ -47,6 +63,11 @@ file in each repository.
 Prometheus is released under the
 [Apache 2.0](https://github.com/prometheus/prometheus/blob/master/LICENSE) license.
 
+### What is the plural of Prometheus?
+
+After extensive research it has been determined that the correct plural of
+'Prometheus' is 'Prometheis'.
```
Comment (juliusv) [inline content/docs/introduction/faq.md]: I thought the conclusion was more nuanced/ambivalent, and it depends on assumptions and preferences. I'd leave this section out for now. If we want to write it properly, we need @beorn7  :)

File: content/docs/introduction/faq.md (hunk 5 above)
Comment (juliusv) [inline content/docs/introduction/faq.md]: Oh right, didn't notice that.

File: content/docs/introduction/faq.md (hunk 6 above)
Comment (beorn7) [inline content/docs/introduction/faq.md]: It's perhaps a nice Easter egg to leave it like this for now. It's correct, but without context, it doesn't make much sense. If people start to ask meta-FAQ about this FAQ, we can add the full context "upon popular request".

File: content/docs/introduction/faq.md (hunk 5 above)
Comment (brian-brazil) [inline content/docs/introduction/faq.md]: Good catch, fixed.

File: content/docs/introduction/faq.md (hunk 6 above)
Comment (juliusv) [inline content/docs/introduction/faq.md]: Oook.

**Pull request merged**